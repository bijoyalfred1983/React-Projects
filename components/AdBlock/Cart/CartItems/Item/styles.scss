@import '../../../../theme/variables.scss';

button:active{
  background: none;
}

button:focus{
  background: none;
}

.estimatedDelivery {
  @media #{$medium-and-up} {
    font-size: 12px;
    line-height: 16px;
    color: $grey-ultra-dark;
    float: left;
    padding: 5px 0;
  }

  @media #{$small-and-down} {
    font-size: 14px;
    line-height: 18px;
    color: $color-green;
  }
}

.partNumber {
  @media #{$small-and-down} {
    padding-left: 10px;
  }
}

@media #{$extra-large-and-up} {
  .mobile {
    display: none !important;
  }

  .exclusion img {
    float: right;
    padding:2px 10px;
  }

  .cardBlock {
    display: flex;
    position: relative;
    flex-flow: column;
    background-color: $color-white-main;
    box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
    margin-bottom: 8px;
    padding-top: 19px;
  }

  .errorText {
    color: $autozone-red;
    padding: 10px 20px;
    width:85%;

    img {
      float:left;
      padding-right: 10px;
    }
  }

  .remove {
    position: absolute;
    top: 16px;
    right: 16px;
    font-size: 14px;
    color: $black-bg-color;
    cursor: pointer;
    background: none;
    border: 0;
    text-decoration: underline;
    font-weight: 500;
    font-family:$font-az-app;
  }

  .header {
    height: 42px;
    padding-left: 22px;
  }

  .itemName {
    display: inline-block;
    font-family: $font-az-medium;
    line-height: 24px;
    text-transform: uppercase;
    font-size: 20px;
    font-weight: bold;
    vertical-align: middle;
  }

  .nextDayDelivery {
    height: 24px;
    width: 154px;
    margin-left: 16px;
    vertical-align: middle;
    display: inline-block;
  }

  .quickNote {
    font-family: $font-az;
    line-height: 18px;
    font-size: 14px;
    color: $az-crust-grey;
  }

  .content {
    height: 126px;
    padding-left: 22px;
    padding-top: 24px;
    display: flex;
    flex-direction: row;
    width: 100%;
  }

  .thumbnail {
    height: 70px;
    width: 70px;
    margin-right: 21px;
  }

  .footer {
    height: 40px;
    border-top: 1px solid $color-lighter-grey;
    padding-right: 20px;
    padding-left: 19px;
    display: flex;
    align-items: center;
    flex-direction: row;
  }

  .seeProductNotes {
    display: block;
    float: right;
    cursor: pointer;
    font-family: $font-az-app;
    font-size: 10px;
    text-transform: uppercase;
    color: $grey-ultra-dark;
    // margin-bottom: 24px;
  }

  .notes {
    border-top: 1px solid $color-lighter-grey;
    font-family: $font-az-app;
    padding: 29px 40px 32px 43px;
    font-size: 14px;
    line-height: 20px;
    color: $az-crust-grey;

    td {
      vertical-align: top;
      padding: 0 0 10px 0;
    }

    td:first-child {
      font-weight: bold;
      width: 146px;
    }
  }

  .itemInfo {
    display: inline-block;
    line-height: 20px;
    font-size: 12px;
    font-family: $font-az-app;
    color: $az-crust-grey;
    margin-bottom: 10px;
  }

  .itemInfoValue,
  .rebate {
    font-weight: bold;
  }

  .rebate {
    text-decoration: underline;
    cursor: pointer;

    a {
      color: $az-crust-grey;
    }
  }
}

.cartLeft {
  border-left: 2px solid $autozone-red;
}

@media #{$extra-large-and-down} {
  .counterpart {
    width: 164px;
    padding: 7px 16px;
    border: 1px solid #000;
  }

  .errorText {
    padding:12px;

    span {
      float:right;
      width: 90%;
    }
  }

  .exclusionMobile {
    font-size: 12px;
  }

  .exclusionMobile img {
    vertical-align: middle;
  }

  .desktop {
    display: none !important;
  }

  .cardBlock {
    display: flex;
    position: relative;
    flex-flow: column;
    background-color: $color-white-main;
    box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
    margin-bottom: 8px;

    @media #{$medium-only} {
      border: 1px solid $color-lighter-grey;
    }
  }

  .aboveHeader {
    margin: 13px 16px 0 16px;
    padding-bottom: 10px;
    border-bottom: 1px solid $color-lighter-grey;
  }

  .nextDayDelivery {
    height: 24px;
    width: 154px;
    display: inline-block;
  }

  .rebate {
    font-size: 12px;
    line-height: 16px;
    color: $az-light-grey;
  }

  .content {
    display: flex;
    padding: 18px 0 0;
    flex-direction: column;
    width: 100%;
  }

  .header {
    padding-left: 16px;
    padding-top: 16px;

    button {
      background: none;
      border: 0;
      float: right;

      @media #{$extra-large-and-down} {
        -webkit-tap-highlight-color: transparent;
      }

      img {
        float: right;
      }
    }
  }

  .itemName {
    display: block;
    width: 252px;
    font-family: $font-az-big-container;
    line-height: 20px;
    text-transform: uppercase;
    font-size: 18px;
    float: left;
    margin-top: 5px;
  }

  .quickNote {
    display: block;
    width: 255px;
    font-family: $font-az;
    line-height: 18px;
    font-size: 14px;
    color: $az-crust-grey;
  }

  .details {
    display: flex;
    margin-top: 24px;
    margin-bottom: 24px;
  }

  .thumbnail {
    height: 84px;
    width: 84px;
  }

  .itemInfo {
    display: inline-block;
    line-height: 18px;
    font-size: 14px;
    font-family: $font-az-app;
    color: $az-crust-grey;
  }

  .warranty {
    text-decoration: underline;
    cursor: pointer;
    color: $grey-ultra-dark;
  }

  .itemInfoValue {
    font-weight: bold;
  }

  .seeProductNotes {
    cursor: pointer;
    font-size: 12px;
    text-decoration: underline;
    color: $az-warm-grey;
    margin-top: 16px;
    margin-bottom: 24px;
  }

  .footer {
    height: 47px;
    border-top: 1px solid $color-lighter-grey;
    margin-right: 16px;
    margin-left: 16px;
    display: block;
    align-items: center;
    flex-direction: row;
  }

  .popup {
    position: fixed;
    background-color: $color-white-main;
    z-index: 2;
    width: 100%;
    top: 0;
    left: 0;
    display: flex;
    flex-flow: column;
    height: 100%;

    .header {
      width: 100%;
      padding: 80px 0 0 16px;
      background-color: $grey-ultra-dark;
    }

    .backButton {
      cursor: pointer;
      display: inline-block;
      height: 40px;
      line-height: 40px;
      text-transform: uppercase;
      color: $color-white-main;
      font-size: 12px;
    }

    .content {
      padding: 16px;
    }

    .itemName {
      color: $black-bg-color;
      width: 100%;
    }

    .quickNote {
      width: 100%;
    }

    .divider {
      width: 100%;
      height: 2px;
      margin-top: 16px;
      background-color: $color-lighter-grey;
    }

    .noteContainer {
      display: flex;
      font-size: 14px;
      line-height: 20px;
      color: $az-crust-grey;
      position: relative;
      padding-bottom: 16px;

      .notes {
        overflow-y: auto;
        -ms-overflow-style: none;
        overflow: -moz-scrollbars-none;
        flex: 1;
      }

      .notes::-webkit-scrollbar {
        width: 0 !important
      }

      .note {
        display: block;
      }
    }

    .shadowCoverTop {
      height: 5px;
      width: 100%;
      background-color: $color-white-main;
      background-size: 100% 12px;
      background-repeat: no-repeat;
      position: relative;
      z-index: 2;
    }

    .shadowContainer {
      position: absolute;
      top: 0;
      left: 0;
      width: 100%;
      height: 100%;
      pointer-events: none;
    }

    .shadow {
      position: absolute;
      z-index: 1;
    }

    .linearShadowTop {
      top: 0;
      left: 0;
      background: linear-gradient(rgba(0, 0, 0, 0.2) 10%, rgba(255, 255, 255, 0)) 100% 0;
      background-repeat: no-repeat;
      background-size: 100% 3px;
      width: inherit;
      height: 4px;
    }
  }
}

.vehicle {
  font-size: 12px;
  display: table;

  img {
    display: table-cell;
    vertical-align: middle;
  }

  .fits {
    color: $color-green;
    display: table-cell;
    vertical-align: middle;
    padding: 0 5px 0 5px;
  }

  .vehicleDescription {
    color: $az-light-grey;
    display: table-cell;
    vertical-align: middle;
  }
}

.price {
  font-family: $font-prod-style;
  font-size: 26px;
  font-weight: 900;
  font-style: normal;
  font-stretch: condensed;
  line-height: 1.08;
  letter-spacing: 0.2px;
  text-align: right;
  color: $black-color;
}

.priceSub {
  font-family: $font-prod-style;
  vertical-align: top;
  font-size: 16px;
  padding: 0;
  font-weight: 900;
  color: $black-color;
}

.corePrice {
  margin-top: 7px;
  font-size: 12px;

  @media (max-width: 767px) {
    text-align: center;
  }
}

.was {
  font-size: 12px;
  font-weight: normal;
  font-style: normal;
  font-stretch: normal;
  line-height: 1.17;
  letter-spacing: normal;
  color: $color-grey;

  @media (max-width: 767px) {
    text-align: center;
  }

  .strike{
    text-decoration: line-through;
  }
}

.pricePart {
  margin-top: 5px;

  @media (max-width: 767px) {
    text-align: center;
  }
}

@media print{
  .remove{
    font-size: 14px;
    color: #000;
    background: none;
    border: 0;
    text-decoration: underline;
    font-weight: 500;
    float:right;
  }

  .mobile{
    display: none !important;
  }

  .cardBlock{
    background: #fff;
    padding-left: 20px;
  }

  .printView{
    float: left !important;
  }

  .printViewContainer {
    width: 33% !important;
  }

  .itemInfo{
    padding-top: 24px;
  }

  .printViewDelivery{
    width: 26% !important;
  }

  .printViewCounter{
    width: 20% !important;
  }

  .printViewPrice{
    width: 15% !important;
  }
}
