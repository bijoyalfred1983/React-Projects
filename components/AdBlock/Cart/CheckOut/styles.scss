@import '../../../theme/variables';

.checkOutContainer{
  width:78%;
  margin-left: auto;  
  margin-right: auto;
  display: flex;
  flex-direction: column;
  position: sticky;
  top: 1px; 

  @media #{$extra-large-and-down} {
    width:100%;
    flex-direction: row-reverse;
    justify-content: space-around;
    margin-top: 10px;
  }

  form {
    @media #{$small-and-down} {
      width: 100%;
    }
  }
}

.totalContainer{
  width:78%;
  margin-left: auto;  
  margin-right: auto;
  display: flex;
  flex-direction: column;
  position: sticky;
  top: 1px; 

  @media #{$extra-large-and-down} {
    width:100%;
    flex-direction: row-reverse;
    justify-content: space-between;
    margin-top: 15px;
  }
}

.bottomSection {
  font-size: 14px;
  font-family: $font-az-cond;
  font-weight: bold;
}

.checkOutButton{
  background-color: $orange-primary;
  color: $color-white-main;
  font-family: $font-az-medium;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: 18px;
  height: 40px;
  cursor: pointer;

  @media #{$extra-large-and-down} {
    width: 146px;
  }

  @media #{$small-and-down} {
    width: 98%;
  }
}

.checkOutButton:hover{
  color:$color-white-main;
}

.paypalButton{
  background-color: $az-checkout-light-grey;
  width: 100%;
  height: 40px;
  display: flex;
  align-items: center;
  justify-content: center;
  border: none;
  cursor: pointer;
  
  @media #{$extra-large-and-down} {
    width: 146px;
  }

  @media #{$small-and-down} {
    width: 95%;
  }
}

.paypalButton:focus{
  background-color: $az-checkout-light-grey;  
}

.checkOutSeparator{
  color: $az-light-grey;
  font-size: 14px;
  font-family: $font-az-app;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 38px;

  @media #{$extra-large-and-down} {
    display: none;
  }
}



