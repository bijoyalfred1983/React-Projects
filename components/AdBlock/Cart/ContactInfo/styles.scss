
@import '../../../theme/variables';

.contactInfoContainer {
  width: 100%;
  padding: 59px 0;
  
  .questionTitle {
    font-family: $font-az-medium;
    font-size: 16px;
    font-weight: 500;
    font-style: normal;
    font-stretch: condensed;
    line-height: 1.2;
    letter-spacing: normal;
    text-transform: uppercase;
  }

  .contactInfo {
    font-family: $font-az-app;
    font-size: 14px;
    font-weight: normal;
    font-style: normal;
    font-stretch: normal;
    line-height: normal;
    letter-spacing: 0.1px;
    margin-top: 5px;
  }

  .contactLinkText {
    text-decoration: underline;
    color: $black-bg-color;
  }
}

@media #{$extra-large-and-down} {
  .contactInfoContainer {
    text-align: center;
    padding: 40px 0;
    background-color: $color-white-main;
    margin-top: 24px;
	
    .questionTitle {
      width: 288px;
      font-size: 20px;   
      margin: auto;   
      color: $grey-ultra-dark;
    }

    .contactInfo {
      width: 288px;
      margin: auto;   
      line-height: 1.43;
      color: $grey-ultra-dark;
    }

    .contactLinkText {
      text-decoration: none;
      color: $grey-ultra-dark;
    }
  }
}

@media #{$medium-only} {
  .contactInfoContainer {
    background-color: $az-grey-light;	
    
	
    .questionTitle {
      width: auto;
      padding-left: 20px;
      text-align: left;	  
    }
	
    .contactInfo {
      width: auto;
      padding-left: 20px;
      text-align: left;
    }
  }
}
