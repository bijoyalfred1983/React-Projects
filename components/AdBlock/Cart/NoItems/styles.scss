@import '../../../theme/variables.scss';

.container {
  display: flex;
  justify-content: center;
  align-items: center;
  flex-direction: column;
  height: 323px;

  @media #{$small-and-down} {
    background: $color-white-main;
  }

  @media #{$medium-and-up} {
    background: $color-light-gray;
  }
}

.cartIsEmpty {
  font-family: $font-az-medium;
  color: $az-warm-grey;
  font-size: 24px;
  line-height: 24px;
  text-transform: uppercase;
  margin-bottom: 15px;
  font-weight: 500;
}

.searchProducts {
  font-size: 16px;
  color: $az-warm-grey;
  font-family: $font-category-cont;
  font-weight: 500;

  @media #{$small-and-down} {
    margin-bottom: 30px;
  }

  @media #{$medium-and-up} {
    margin-bottom: 39px;
  }
}

.homeButton {
  font-family: $font-az-medium;
  text-transform: uppercase;
  background: $black-bg-color;
  color: $color-white-main;
  padding: 11px 20px;
}
