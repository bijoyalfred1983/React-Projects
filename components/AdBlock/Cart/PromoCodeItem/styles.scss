@import '../../../theme/variables';

.promoCodeContainer {
  .promoCodeCard {
    display: flex;
    align-items: center;
    justify-content: space-between;
    background-color: $color-white-main;
    box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.2);
    margin: 24px 0;
    padding: 16px;
  }

  .promoTextContainer {
    display: flex;
    align-items: center;
  }

  .promoCartImg {
    width: 48px;
    height: 48px;
    margin-right: 12px;
  }

  .promoTitleText {
    font-family: $font-az-medium;
    font-size: 20px;
    font-weight: 500;
    font-style: normal;
    font-stretch: condensed;
    line-height: 1.2;
    letter-spacing: normal;
    color: $az-dark-grey-text;
    width: 416px;
    flex: 1;
  }

  .couponText {
    color: $orange-primary;      
  }

  .applyButton {
    width: 160px;
    object-fit: contain;
    font-family: $font-az-medium !important;
    border: solid 1px $az-crust-grey !important;
    box-shadow: none !important;
    border-radius: 0 !important;
    font-size: 15px;
    font-weight: 500;
    font-style: normal;
    font-stretch: condensed;
    line-height: normal;
    letter-spacing: normal;
    color: $black-bg-color !important;

    @media #{$extra-large-and-up} {
      height: 40px !important;
    }

    @media #{$extra-large-and-down} {
      width: 120px;
    }

    &:hover {
      border-bottom: 3px;

      div {
        @media #{$extra-large-and-up} {
          height: 40px !important;
        }
      }
    }
  }
}

@media #{$extra-large-and-up} {
  .promoCodeCardMobile {
    display: none;
  }
}

@media #{$medium-only} {
  .promoCodeContainer {
    padding: 0;
    margin: 0 10px;
    border: 1px solid $crust-white-color;
  }
}

@media #{$extra-large-and-down} {
  .promoCodeContainer {
    padding: 0 16px 16px;
    background-color: $color-white-main;

    .promoCodeCard {
      display: none;
    }

    .promoCodeCardMobile {
      max-width: 320px;
      width: 100%;
      margin: auto;
    }

    .promoTitleText {
      font-family: $font-az-app;
      font-size: 12px;
      font-weight: 500;
      font-style: normal;
      font-stretch: normal;
      line-height: 1.67;
      letter-spacing: normal;
      color: $grey-ultra-dark;
      width: 100%;
      text-align: center;
    }

    .persentageText {
      color: $color-green;        
    }

    .promoTextContainer {
      justify-content: space-around;
      margin-top: 7px;
    }

    .couponTextContent {
      width: 160px;
      height: 36px;
      border: 1px dashed $az-crust-grey;
      margin-right: 8px;
    }

    .couponText {
      font-family: $font-az-medium;
      font-size: 20px;
      font-weight: 500;
      text-align: center;
      height: 34px;
      line-height: 34px;
      color: $black-bg-color;
    }
  }
}

@media print{
  .promoCodeCardMobile{
    display:none;
  }
}
