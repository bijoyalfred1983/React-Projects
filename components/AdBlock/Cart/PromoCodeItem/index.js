/* @flow */
import React from 'react';
import RaisedButton from 'material-ui/RaisedButton';
import styles from './styles.scss';

type Props = {
  staticLabelData: Object
};
const promoComponentDesktop = (props: Props.staticLabelData) => (
  <div className={styles.promoCodeCard}>
    <span className={styles.promoTextContainer}>
      <img
        src="/images/cart-star.png"
        alt="cart logo"
        className={styles.promoCartImg}
      />
      <span className={styles.promoTitleText}>
        {props !== undefined && props.label_cart_promo_Receive
          ? props.label_cart_promo_Receive.toUpperCase()
          : 'label_cart_promo_Receive'}
        {' 20% '}
        {props !== undefined &&
        props.label_cart_promo_OffShipToHomeItemsWithOurPromoCode
          ? props.label_cart_promo_OffShipToHomeItemsWithOurPromoCode.toUpperCase()
          : 'label_cart_promo_OffShipToHomeItemsWithOurPromoCode'}{' '}
        <span className={styles.couponText}>{'AutoZone123'.toUpperCase()}</span>
      </span>
    </span>
    <RaisedButton
      label={
        props !== undefined && props.label_cart_promo_ApplyNow
          ? props.label_cart_promo_ApplyNow
          : 'label_cart_promo_ApplyNow'
      }
      className={styles.applyButton}
    />
  </div>
);

const promoComponentMobile = (props: Props.staticLabelData) => (
  <div className={styles.promoCodeCardMobile}>
    <div className={styles.promoTitleText}>
      <strong>
        {props !== undefined && props.label_cart_promo_Congrats
          ? props.label_cart_promo_Congrats
          : 'label_cart_promo_Congrats'}
      </strong>
      {props !== undefined && props.label_cart_promo_YouQualifyFor
        ? props.label_cart_promo_YouQualifyFor
        : 'label_cart_promo_YouQualifyFor'}
      <span className={styles.persentageText}>
        <strong>20%</strong>
      </span>
      {props !== undefined && props.label_cart_promo_OffShipToHome
        ? props.label_cart_promo_OffShipToHome
        : 'label_cart_promo_OffShipToHome'}
    </div>
    <div className={styles.promoTextContainer}>
      <div className={styles.couponTextContent}>
        <p className={styles.couponText}>AUTOZONE123</p>
      </div>
      <RaisedButton
        label={
          props !== undefined && props.label_cart_orderSummary_Apply
            ? props.label_cart_orderSummary_Apply
            : 'label_cart_orderSummary_Apply'
        }
        className={styles.applyButton}
      />
    </div>
  </div>
);

export default (props: Props) => (
  <div className={styles.promoCodeContainer}>
    {promoComponentDesktop(props.staticLabelData)}
    {promoComponentMobile(props.staticLabelData)}
  </div>
);
