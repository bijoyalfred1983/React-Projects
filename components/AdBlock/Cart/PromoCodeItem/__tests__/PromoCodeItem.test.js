import React from 'react';
import { mount } from 'enzyme';
import { MuiThemeProvider } from 'material-ui/styles';
import RaisedButton from 'material-ui/RaisedButton';
import toJson from 'enzyme-to-json';
import PromoCodeItem from '../';

describe('<PromoCodeItem />', () => {
	let PROPS;
	PROPS = {
      staticLabelData: { test: 'aa' }
  }
  test('renders correctly PromoCodeItem', () => {
    const wrapper = mount(
      <MuiThemeProvider>
        <PromoCodeItem {...PROPS}/>
      </MuiThemeProvider>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
