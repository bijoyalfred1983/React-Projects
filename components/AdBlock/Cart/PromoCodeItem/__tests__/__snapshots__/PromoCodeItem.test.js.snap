// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PromoCodeItem /> renders correctly PromoCodeItem 1`] = `
<MuiThemeProvider>
  <Component>
    <div
      className="promoCodeContainer"
    >
      <div
        className="promoCodeCard"
      >
        <span
          className="promoTextContainer"
        >
          <img
            alt="cart logo"
            className="promoCartImg"
            src="/images/az-cart.png"
          />
          <span
            className="promoTitleText"
          >
            RECEIVE 20% OFF SHIP TO HOME ITEMS WITH OUR PROMO CODE 
            <span
              className="couponText"
            >
              AUTOZONE123
            </span>
          </span>
        </span>
        <RaisedButton
          className="applyButton"
          disabled={false}
          fullWidth={false}
          label="APPLY NOW"
          labelPosition="after"
          primary={false}
          secondary={false}
        >
          <Paper
            circle={false}
            className="applyButton"
            rounded={true}
            style={
              Object {
                "display": "inline-block",
                "minWidth": 88,
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              }
            }
            transitionEnabled={true}
            zDepth={1}
          >
            <div
              className="applyButton"
              style={
                Object {
                  "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                  "backgroundColor": "#ffffff",
                  "borderRadius": 2,
                  "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
                  "boxSizing": "border-box",
                  "color": "rgba(0, 0, 0, 0.87)",
                  "display": "inline-block",
                  "fontFamily": "Roboto, sans-serif",
                  "minWidth": 88,
                  "muiPrepared": true,
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                }
              }
            >
              <EnhancedButton
                containerElement="button"
                disabled={false}
                focusRippleColor="rgba(0, 0, 0, 0.87)"
                focusRippleOpacity={0.1}
                onBlur={[Function]}
                onClick={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onKeyboardFocus={[Function]}
                onMouseDown={[Function]}
                onMouseEnter={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchStart={[Function]}
                style={
                  Object {
                    "backgroundColor": "#ffffff",
                    "borderRadius": 2,
                    "height": 36,
                    "lineHeight": "36px",
                    "padding": 0,
                    "textAlign": "center",
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                    "width": "100%",
                  }
                }
                tabIndex={0}
                touchRippleColor="rgba(0, 0, 0, 0.87)"
                touchRippleOpacity={0.1}
                type="button"
              >
                <button
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  style={
                    Object {
                      "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                      "backgroundColor": "#ffffff",
                      "border": 10,
                      "borderRadius": 2,
                      "boxSizing": "border-box",
                      "cursor": "pointer",
                      "display": "inline-block",
                      "fontFamily": "Roboto, sans-serif",
                      "fontSize": "inherit",
                      "fontWeight": "inherit",
                      "height": 36,
                      "lineHeight": "36px",
                      "margin": 0,
                      "muiPrepared": true,
                      "outline": "none",
                      "padding": 0,
                      "position": "relative",
                      "textAlign": "center",
                      "textDecoration": "none",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "verticalAlign": null,
                      "width": "100%",
                    }
                  }
                  tabIndex={0}
                  type="button"
                >
                  <TouchRipple
                    abortOnScroll={true}
                    color="rgba(0, 0, 0, 0.87)"
                    key="touchRipple"
                    opacity={0.1}
                  >
                    <div
                      onMouseDown={[Function]}
                      onMouseLeave={[Function]}
                      onMouseUp={[Function]}
                      onTouchEnd={[Function]}
                      onTouchStart={[Function]}
                    >
                      <div
                        style={
                          Object {
                            "backgroundColor": false,
                            "borderRadius": 2,
                            "height": 36,
                            "muiPrepared": true,
                            "top": 0,
                            "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                          }
                        }
                      >
                        <span
                          key="labelElement"
                          style={
                            Object {
                              "color": "rgba(0, 0, 0, 0.87)",
                              "fontSize": 14,
                              "fontWeight": 500,
                              "letterSpacing": 0,
                              "margin": 0,
                              "muiPrepared": true,
                              "opacity": 1,
                              "paddingLeft": 16,
                              "paddingRight": 16,
                              "position": "relative",
                              "textTransform": "uppercase",
                              "userSelect": "none",
                            }
                          }
                        >
                          APPLY NOW
                        </span>
                      </div>
                    </div>
                  </TouchRipple>
                </button>
              </EnhancedButton>
            </div>
          </Paper>
        </RaisedButton>
      </div>
      <div
        className="promoCodeCardMobile"
      >
        <div
          className="promoTitleText"
        >
          <strong>
            Congrats!
          </strong>
           You qualify for 
          <span
            className="persentageText"
          >
            <strong>
              20%
            </strong>
          </span>
           off ship-to-home
        </div>
        <div
          className="promoTextContainer"
        >
          <div
            className="couponTextContent"
          >
            <p
              className="couponText"
            >
              AUTOZONE123
            </p>
          </div>
          <RaisedButton
            className="applyButton"
            disabled={false}
            fullWidth={false}
            label="APPLY"
            labelPosition="after"
            primary={false}
            secondary={false}
          >
            <Paper
              circle={false}
              className="applyButton"
              rounded={true}
              style={
                Object {
                  "display": "inline-block",
                  "minWidth": 88,
                  "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                }
              }
              transitionEnabled={true}
              zDepth={1}
            >
              <div
                className="applyButton"
                style={
                  Object {
                    "WebkitTapHighlightColor": "rgba(0,0,0,0)",
                    "backgroundColor": "#ffffff",
                    "borderRadius": 2,
                    "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
         0 1px 4px rgba(0, 0, 0, 0.12)",
                    "boxSizing": "border-box",
                    "color": "rgba(0, 0, 0, 0.87)",
                    "display": "inline-block",
                    "fontFamily": "Roboto, sans-serif",
                    "minWidth": 88,
                    "muiPrepared": true,
                    "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                  }
                }
              >
                <EnhancedButton
                  containerElement="button"
                  disabled={false}
                  focusRippleColor="rgba(0, 0, 0, 0.87)"
                  focusRippleOpacity={0.1}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onKeyboardFocus={[Function]}
                  onMouseDown={[Function]}
                  onMouseEnter={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchStart={[Function]}
                  style={
                    Object {
                      "backgroundColor": "#ffffff",
                      "borderRadius": 2,
                      "height": 36,
                      "lineHeight": "36px",
                      "padding": 0,
                      "textAlign": "center",
                      "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                      "width": "100%",
                    }
                  }
                  tabIndex={0}
                  touchRippleColor="rgba(0, 0, 0, 0.87)"
                  touchRippleOpacity={0.1}
                  type="button"
                >
                  <button
                    disabled={false}
                    onBlur={[Function]}
                    onClick={[Function]}
                    onFocus={[Function]}
                    onKeyDown={[Function]}
                    onKeyUp={[Function]}
                    onMouseDown={[Function]}
                    onMouseEnter={[Function]}
                    onMouseLeave={[Function]}
                    onMouseUp={[Function]}
                    onTouchEnd={[Function]}
                    onTouchStart={[Function]}
                    style={
                      Object {
                        "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                        "backgroundColor": "#ffffff",
                        "border": 10,
                        "borderRadius": 2,
                        "boxSizing": "border-box",
                        "cursor": "pointer",
                        "display": "inline-block",
                        "fontFamily": "Roboto, sans-serif",
                        "fontSize": "inherit",
                        "fontWeight": "inherit",
                        "height": 36,
                        "lineHeight": "36px",
                        "margin": 0,
                        "muiPrepared": true,
                        "outline": "none",
                        "padding": 0,
                        "position": "relative",
                        "textAlign": "center",
                        "textDecoration": "none",
                        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                        "verticalAlign": null,
                        "width": "100%",
                      }
                    }
                    tabIndex={0}
                    type="button"
                  >
                    <TouchRipple
                      abortOnScroll={true}
                      color="rgba(0, 0, 0, 0.87)"
                      key="touchRipple"
                      opacity={0.1}
                    >
                      <div
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchStart={[Function]}
                      >
                        <div
                          style={
                            Object {
                              "backgroundColor": false,
                              "borderRadius": 2,
                              "height": 36,
                              "muiPrepared": true,
                              "top": 0,
                              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                            }
                          }
                        >
                          <span
                            key="labelElement"
                            style={
                              Object {
                                "color": "rgba(0, 0, 0, 0.87)",
                                "fontSize": 14,
                                "fontWeight": 500,
                                "letterSpacing": 0,
                                "margin": 0,
                                "muiPrepared": true,
                                "opacity": 1,
                                "paddingLeft": 16,
                                "paddingRight": 16,
                                "position": "relative",
                                "textTransform": "uppercase",
                                "userSelect": "none",
                              }
                            }
                          >
                            APPLY
                          </span>
                        </div>
                      </div>
                    </TouchRipple>
                  </button>
                </EnhancedButton>
              </div>
            </Paper>
          </RaisedButton>
        </div>
      </div>
    </div>
  </Component>
</MuiThemeProvider>
`;
